packages/titus-infra-aws-mira/package-lock.json

```diff
@@ -8242,7 +8242,8 @@
       "version": "1.3.0",
       "resolved": "https://registry.npmjs.org/growly/-/growly-1.3.0.tgz",
       "integrity": "sha1-8QdIy+dq+WS3yWyTxrzCivEgwIE=",
-      "dev": true
+      "dev": true,
+      "optional": true
```

Comment:
This is getting automatically added when I run either `npm i` or `npm ci`. I'm on npm@6. With npm@7 all package-locks are heavily reformatted/changed.

Reply:
not a problem